### PR TITLE
FIX(Urls): Class Urls is not exported from _reactor.py to reactor.py in Qpid Proton Python

### PR DESCRIPTION
**Pull request by rkubis**
…tor.py in Qpid Proton Python

Relates:
  PROTON-1997 [Python] Urls class is not exported by reactor

### CODE REVIEW

**Comment by pematous** [comment]
I don't like this. This is an existing bug in the library that was recently introduced. It should get fixed and I don't want to introduce workarounds to the master. This introduce risks to forget the roll-back after fixed.

@michalxo If you need a working pip package you should probably create your own and use local-install.

**Comment by ssorj** [comment]
@rkubis, what is the role of Urls in the cli tool code?  It doesn't look to me like it's doing anything more than a list of strings would do, and a list of strings is the documented interface to connect.

I *think* Urls was an implementation detail that was previously exposed by accident.

**Comment by rkubis** [comment]
@ssorj Thanks for note, I will investigate the proposed solution.

**Comment by jdanekrh** [comment]
@astitcher Hi, in case you read GH notifications :P Could you please check https://issues.apache.org/jira/browse/PROTON-1997 and confirm if the move of `Urls` was intentional or not? We could deal with PROTON-1997 and this PR if we knew that for sure.

**Comment by astitcher** [comment]
To (try to) clarify what Justin is saying:
As far as I can tell, you don't need the `Urls` class *at all*. It is a purely internal detail that was exposed accidentally and is not part of the API to the reactor connect method.
You can replace the line with `conn_opts['urls'] = self.opts.conn_urls` and that should fix the problem and work as before.

File: cli_proton_python/coreclient.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def parse_connection_options(self):
             if not isinstance(self.opts.conn_urls, (str, list)):
                 raise ValueError('Invalid conn-urls value, expected string or list: %s'
                                  % self.opts.conn_urls)
-            conn_opts['urls'] = proton.reactor.Urls(self.opts.conn_urls)
+            conn_opts['urls'] = proton._reactor.Urls(self.opts.conn_urls)
```
Comment (jdanekrh) [inline cli_proton_python/coreclient.py]:
```suggestion
            conn_opts['urls'] = self.opts.conn_urls
```

**Pull request merged**